README.md

```diff
@@ -36,6 +36,13 @@ repos:
     # Check if minimize id changes is selected in the plc project file.
     # See https://www.youtube.com/watch?v=KKpBtaYjfWo&t=935s why to do this.
     -   id: minimize-id-changes
+    # Checks if TwinCAT versions match in different tsproj files, or if it matches the targeted one.
+    -   id: check-twincat-versions
+        # Possible optional arguments
+        # --target-version: Set a version that you want the tsproj file to have
+        # --fix: Fix the version numbers if a target version is set
+        # --reason: Add a reason to the error message in case of a non-matching version.
+        args: [--target-version=3.1.4024.20, --fix, --reason="This version has a crucial new feature"]
```

Comment:
We should document the pinning argument too. Possibly this is the intent, given the BooleanOptionalAction
```suggestion
        # --reason: Add a reason to the error message in case of a non-matching version.
        # --pinned: Require the TwinCAT version to be pinned. Apply pinning if combined with --fix
        # --no-pinned: Require the TwinCAT version to not be pinned. Remove pinning of combined with --fix
        args: [--target-version=3.1.4024.20, --fix, --reason="This version has a crucial new feature"]
```